rewrite-java/src/main/java/org/openrewrite/java/OrderImports.java

```diff
@@ -60,7 +60,9 @@ public String getDisplayName() {
 
     @Override
     public String getDescription() {
-        return "Group and order imports.";
+        return "Groups and orders import statements. If a style has been defined, this recipe will order the imports " +
```

Comment:
How about a Markdown hyperlink to the docs? https://docs.openrewrite.org/concepts-explanations/styles
```suggestion
        return "Groups and orders import statements. If a [style has been defined](https://docs.openrewrite.org/concepts-explanations/styles), this recipe will order the imports " +
```

Reply:
Great suggestion!